src/js/components/DataTable/README.md

```diff
@@ -447,7 +447,12 @@ When supplied, causes checkboxes to be added to each row such that
       will be called with an array of primary key values, suitable to be
       passed to the 'select' property. If you are storing select state via
       a 'useState' hook, you can do something like:
-      '<DataTable select={select} onSelect={setSelect} />'.
+      '<DataTable select={select} onSelect={setSelect} />'. The rendering of the
+      selected column is generated automatically, if you like to customize its 
+      column size or align properties, add the reserved property name of 
+      'gdt-selected' with the attributes of 
+      {property: 'gdt-selected', size: '0px', align: 'end' } 
```

Comment:
Hmm... This feels a bit hack-y to me.

1. I'm thinking we should change the default behavior to restrict the size of the cell containing the selection checkbox to never grow or shrink.
2. If we end up customizing via `columns`, I think we should omit `property` and use something like `selection: true`.

Reply:
OK. I'll close this PR in this case.